tests/decorators/decorators.go

```diff
@@ -24,23 +24,24 @@ var (
 	KSMRequired = []interface{}{Label("KSM-required")}
 
 	// Features
-	Sysprep                     = []interface{}{Label("Sysprep")}
-	Windows                     = []interface{}{Label("Windows")}
-	Networking                  = []interface{}{Label("Networking")}
-	VMIlifecycle                = []interface{}{Label("VMIlifecycle")}
-	Expose                      = []interface{}{Label("Expose")}
-	NonRoot                     = []interface{}{Label("verify-non-root")}
-	NativeSsh                   = []interface{}{Label("native-ssh")}
-	ExcludeNativeSsh            = []interface{}{Label("exclude-native-ssh")}
-	Reenlightenment             = []interface{}{Label("Reenlightenment")}
-	TscFrequencies              = []interface{}{Label("TscFrequencies")}
-	PasstGate                   = []interface{}{Label("PasstGate")}
-	VMX                         = []interface{}{Label("VMX")}
-	Upgrade                     = []interface{}{Label("Upgrade")}
-	CustomSELinux               = []interface{}{Label("CustomSELinux")}
-	Istio                       = []interface{}{Label("Istio")}
-	InPlaceHotplugNICs          = []interface{}{Label("in-place-hotplug-NICs")}
-	MigrationBasedHotplugNICs   = []interface{}{Label("migration-based-hotplug-NICs")}
-	RequiresTwoSchedulableNodes = []interface{}{Label("requires-two-schedulable-nodes")}
-	VMLiveUpdateFeaturesGate    = []interface{}{Label("VMLiveUpdateFeaturesGate")}
+	Sysprep                      = []interface{}{Label("Sysprep")}
+	Windows                      = []interface{}{Label("Windows")}
+	Networking                   = []interface{}{Label("Networking")}
+	VMIlifecycle                 = []interface{}{Label("VMIlifecycle")}
+	Expose                       = []interface{}{Label("Expose")}
+	NonRoot                      = []interface{}{Label("verify-non-root")}
+	NativeSsh                    = []interface{}{Label("native-ssh")}
+	ExcludeNativeSsh             = []interface{}{Label("exclude-native-ssh")}
+	Reenlightenment              = []interface{}{Label("Reenlightenment")}
+	TscFrequencies               = []interface{}{Label("TscFrequencies")}
+	PasstGate                    = []interface{}{Label("PasstGate")}
+	VMX                          = []interface{}{Label("VMX")}
+	Upgrade                      = []interface{}{Label("Upgrade")}
+	CustomSELinux                = []interface{}{Label("CustomSELinux")}
+	Istio                        = []interface{}{Label("Istio")}
+	InPlaceHotplugNICs           = []interface{}{Label("in-place-hotplug-NICs")}
+	MigrationBasedHotplugNICs    = []interface{}{Label("migration-based-hotplug-NICs")}
+	RequiresTwoSchedulableNodes  = []interface{}{Label("requires-two-schedulable-nodes")}
+	VMLiveUpdateFeaturesGate     = []interface{}{Label("VMLiveUpdateFeaturesGate")}
+	RequiresRWXFilesystemStorage = []interface{}{Label("requires-rwx-filesystem-storage")}
```

Comment:
I wonder about "scalability" of this label. Will we require "RWO", "ROM" with combination of "block"? The nice thing about using labels is that you can define 2 types of labels: block/fs and rwo,rom,rwx. Then you can specify that you have rwx-fs as rwx&&fs

Reply:
That would fail because rwx just means there is storage that is doing rwx. it may only do rwx block
Shortened the label to just `rwxfs`